### PR TITLE
first pass support for sticker tome

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/resources/2008/Stickers.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+export function currentStickers(): [Item, Item, Item] {
+  return $slots`sticker1, sticker2, sticker3`.map((s) => equippedItem(s)) as [
+    Item,
+    Item,
+    Item
+  ];
+}
+
+export function setStickers(
```
Comment (gausie) [inline src/resources/2008/Stickers.ts:62]: This should either acquire or throw if you don't have them. And check responses from applying them too

Comment (horrible-little-slime) [reply src/resources/2008/Stickers.ts:63]: threw in a retrieveItem call

**Comment by horrible-little-slime** [comment]
I am abandoning ship but leaving this up for someone to scuttle

**Comment by horrible-little-slime** [comment]
Undrafting because I have no recollection as to why this is a draft.

**Review by gausie — APPROVED** [review]

**Pull request merged**